src/Approximations/overapproximate.jl

```diff
@@ -1963,6 +1963,43 @@ function overapproximate(qm::QuadraticMap{N, <:AbstractZonotope}, ::Type{<:Zonot
     return Zonotope(d, remove_zero_columns(h))
 end
 
+"""
+    overapproximate(overapproximate(QM::QuadraticMap, ::Type{SparsePolynomialZonotope}))
+
+Return a sparse polynomial zonotope overapproximation of the quadratic map of a sparse
+polynomial zonotope. Implements Proposition 3.1.31 of [1].
+
+[1] N. Kochdumper. *Extensions of polynomial zonotopes and their application to
+verification of cyber-physical systems*. 2021.
+"""
+function overapproximate(QM::QuadraticMap, ::Type{SparsePolynomialZonotope})
```

Comment:
You need to constrain the `QuadraticMap`'s set type (otherwise this would be incorrect).

```suggestion
"""
    overapproximate(QM::QuadraticMap{N, <:SparsePolynomialZonotope},
                    ::Type{SparsePolynomialZonotope})

Return a sparse polynomial zonotope overapproximation of the quadratic map of a
sparse polynomial zonotope.

### Input

- `QM` -- quadratic map of a sparse polynomial zonotope
- `SparsePolynomialZonotope` -- target type

### Output

A sparse polynomial zonotope overapproximating the quadratic map of a sparse
polynomial zonotope.

### Algorithm

This method implements Proposition 3.1.31 of [1].

[1] N. Kochdumper. *Extensions of polynomial zonotopes and their application to
verification of cyber-physical systems*. 2021.
"""
function overapproximate(QM::QuadraticMap{N, <:SparsePolynomialZonotope},
                         ::Type{SparsePolynomialZonotope})
```